css/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+	font-family: ariel, helvetica, sans-serif;
+	font-size: 18px;
+	line-height: 1.5;
+	margin: 30px;
+	border-bottom: 5px solid #222222;
+	border-top: 5px solid #222222;
+	padding: 50px;
+	background-image: url("../images/mcqueen_white.jpg");
+}
+img.displayed {
```

Comment:
@nyvangord you could have used `.displayed {}` instead, using img.displayed is not really necessary.